docs/docs/README.md

```diff
@@ -4,24 +4,72 @@ id: intro
 slug: /intro
 sidebar_position: 1
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { Gear, Plus } from "@phosphor-icons/react";
 
 
 ![Banner](/img/banner.png)
 
 ---
 
 > Welcome to Butterfly, the opensource note app.
-> Use the sidebar to navigate through the wiki.
+
+Butterfly is a note taking up that is aiming for unified experience across devices. Butterfly's approach is for minimalistic view of the drawing canvas while maximizing the user's control over the app.
 
 ## Getting started
+> You can use the sidebar to navigate quicly through the wiki.
+
+To start working on the app you have two options:
+1. [Downloads](/downloads) the app version that is appropriate for your platform (Butterfly can run on most mobile, desktop, and web platfroms). 
+2. Use [web version](https://butterfly.linwood.dev) if you cannot find your platform or if you don't want to install it on it. See [here](storage#web) about file storage on the web version.
+
+>Once you open the app, continue reading. 
 
-Visit [the downloads](/downloads) to download the app for your platform.
-Use [web version](https://butterfly.linwood.dev) if you cannot find your platform.
 
 ## Main view
 
 ![Main view](main.png)
 
+The main view is composed of several elements
+<Tabs
```

Comment:
this part is not rendering correctly
Fix it

Reply:
I guess, it only renders on web(docs) not on github md preview

Reply:
Yeah, maybe. I started the preview ci, we can look at it if it will be rendered correctly